VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -234,8 +243,8 @@ def post_to_wireserver(self, data):
                 else:
                     raise Exception("no response from encryption settings update request")
             except Exception as e:
+                retry_count += 1
                 if retry_count < retry_count_max:
-                    retry_count += 1
                     time.sleep(5) # sleep for 5 seconds before retrying.
```

Comment:
It would be helpful for log analysis to also log the retry count and the exception message that is triggering the retry within this block.  Otherwise I believe the first 2 exception messages will get swallowed.